pfcpiface/conn.go

```diff
@@ -179,12 +179,26 @@ func (pConn *PFCPConn) setLocalNodeID(id string) {
 
 // Serve serves forever a single PFCP peer.
 func (pConn *PFCPConn) Serve() {
-	go func() {
+	connClosed := make(chan struct{}, 1)
```

Comment:
The name is misleading, it looks like a read timeout can happen even with a connection that is still open
```suggestion
	connTimeout := make(chan struct{}, 1)
```